cmd/operator/main.go

```diff
@@ -181,6 +181,8 @@ func parseFlags(fs *flag.FlagSet) {
 	fs.Var(&cfg.SecretListWatchFieldSelector, "secret-field-selector", "Field selector to filter Secrets to watch")
 	fs.Var(&cfg.SecretListWatchLabelSelector, "secret-label-selector", "Label selector to filter Secrets to watch")
 
+	fs.BoolVar(&cfg.EnableThanosHTTPPort, "enable-thanos-http-port", false, "Whether to enable Thanos HTTP Port")
```

Comment:
What if we expose a CLI flag to set the port number (e.g. `thanos-sidecar-service-port`)? If the value is zero (default) then the operator wouldn't expose the Thanos port.

Reply:
Agreed